tests/units/common/test_enhancer.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from common.enhancer import Enhancer
+from freezegun import freeze_time
+
+
+class PseudoParsesOutput:
+    def get_output(self):
+        return {
+            "abstracts": "this is abstracts",
+            "copyright_holder": "copyright_holder",
+            "copyright_year": "2020",
+            "copyright_statement": "copyright_statement",
+            "copyright_material": "copyright_material",
+            "date_published": "2022-05-20",
+            "title": "title",
+            "subtitle": "subtitle",
+        }
+
+
+@pytest.fixture(scope="function")
+def geneirc_pseudo_parser_output():
+    yield PseudoParsesOutput()
```

Comment:
This should directly return the value instead. `PseudoParsesOutput` class is not needed. Just return the dict present in PseudoParsesOutput.get_ouput directly here